specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -1,2636 +1,2978 @@
 {
- "swagger": "2.0",
- "info": {
-  "version": "1.0",
-  "title": "Face API",
-  "description": "An API for face detection, verification, and identification."
- },
- "securityDefinitions": {
-  "apim_key": {
-   "type": "apiKey",
-   "name": "Ocp-Apim-Subscription-Key",
-   "in": "header"
-  }
- },
- "x-ms-parameterized-host": {
-  "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
-  "parameters": [
-   {
-    "$ref": "../../Common/Parameters.json#/parameters/AzureRegion"
-   }
-  ]
- },
- "basePath": "/face/v1.0",
- "schemes": [
-  "https"
- ],
- "paths": {
-  "/findsimilars": {
-   "post": {
-    "description": "Given query face's faceId, find the similar-looking faces from a faceId array or a faceListId.",
-    "operationId": "Face_FindSimilar",
-    "parameters": [
-     {
-      "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
-     },
-     {
-      "name": "body",
-      "in": "body",
-      "required": true,
-      "x-ms-client-flatten": true,
-      "schema": {
-       "$ref": "#/definitions/FindSimilarRequest"
-      }
-     }
-    ],
-    "consumes": [
-     "application/json"
-    ],
-    "responses": {
-     "200": {
-      "description": "A successful call returns an array of the most similar faces represented in faceId if the input parameter is faceIds or persistedFaceId if the input parameter is faceListId.",
-      "schema": {
-       "$ref": "#/definitions/SimilarFaceResults"
-      },
-      "examples": {
-       "application/json": [
-        {
-         "persistedFaceId": "015839fb-fbd9-4f79-ace9-7675fc2f1dd9",
-         "confidence": 0.82
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Face API",
+        "description": "An API for face detection, verification, and identification."
+    },
+    "securityDefinitions": {
+        "apim_key": {
```

Comment:
I learned today that @fearthecowboy has strong opinions on whitespace and will enforce that we use two-space indentation instead of the four-space indentation seen here.